### DISCUSSION TITLE
Should we remove MDWalters125?

### DISCUSSION THREAD

**Discussion by JoshAtticus** [MeowerBots/.github#2] [Polls]
The copy of MDWalters125 stored in this organisation account is outdated and belongs to @mdwalters. Please vote below on whether we should delete it.

**Comment by showierdata9978** [comment]
I archived it. people should know its outdated

**Reply by JoshAtticus** [reply]
Yes, but it still belongs to md, so we shouldn’t keep it here

**Comment by theycallhermax** [comment]
i still want the original repo back
